srcpkgs/spotifyd/template

```diff
@@ -1,25 +1,36 @@
 # Template file for 'spotifyd'
 pkgname=spotifyd
 version=0.2.24
-revision=1
+revision=2
 build_style=cargo
+configure_args="--no-default-features"
 hostmakedepends="pkg-config"
-makedepends="alsa-lib-devel libressl-devel pulseaudio-devel dbus-devel"
+makedepends="libressl-devel $(vopt_if alsa alsa-devel)
```

Comment:
should be alsa-lib-devl instead of alsa-devel

Reply:
Yup, thanks!